src/arch/host/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(GENERATED_DIRECTORY ${PROJECT_BINARY_DIR}/generated)
 file(MAKE_DIRECTORY ${GENERATED_DIRECTORY}/include)
 
 target_include_directories(sof_options INTERFACE ${PROJECT_SOURCE_DIR}/src/arch/host/include)
-target_include_directories(sof_options INTERFACE ${PROJECT_SOURCE_DIR}/src/library/include)
+target_include_directories(sof_options INTERFACE ${PROJECT_SOURCE_DIR}/src/platform/library/include)
```

Comment:
Please say why this is better in commit message.

Reply:
Done